airflow/models/taskinstance.py

```diff
@@ -3476,8 +3476,13 @@ def _schedule_downstream_tasks(
                     run_id=ti.run_id,
                 ),
                 session=session,
-                nowait=True,
-            ).one()
+                skip_locked=True,
+            ).one_or_none()
+
+            if not dag_run:
+                cls.logger().info("Skip locked rows, rollback")
```

Comment:
I think better slightly reduce severity level. 

```suggestion
                cls.logger().debug("Skip locked rows, rollback")
```

No need to spam into the log for regular users